### ISSUE TITLE
storyboard problem

### ISSUE THREAD

**Issue by odedharth** [sochalewski/UIImageViewAlignedSwift#6]
when I try to add user defined attribute I get this error:

`Failed to set (alignBottom) user defined inspected property on (UIImageViewAlignedSwift.UIImageViewAligned): [<UIImageViewAlignedSwift.UIImageViewAligned 0x7fe046480f00> setValue:forUndefinedKey:]: this class is not key value coding-compliant for the key alignBottom.
`

**Comment by sochalewski** [comment]
I dropped `alignX`properties in Swift version. You should use `alignment` in the code instead.
Take a look at the #4.

**Comment by odedharth** [comment]
@sochalewski thanks for the reply.

I used the alignment in the code, but I've encountered another problem.
I've set the image inside a custom uitableviewcell this way:

```
    @IBOutlet weak var thumb: UIImageViewAligned!
    
    override func awakeFromNib() {
        super.awakeFromNib()
    
        thumb.alignment = .Bottom
        
    } 
```

But after dragging the scroll view of the table a bit this duplication of the image happens:

https://www.evernote.com/l/ADmYqT5xlCFEMaiqtzCQHocBzmLWPsNeU2s 
(in this example the duplication is happening in the first and third cells)
Any idea why is this happening?
